tests/isotp_fast/conformance/src/main.c

```diff
@@ -739,6 +739,40 @@ ZTEST(isotp_fast_conformance, test_sf_length)
 
     check_frame_series(&des_frame, 1, &frame_msgq);
 }
+
+ZTEST(isotp_fast_conformance, test_send_length_63)
+{
+    int ret;
+    struct frame_desired des_frames[2];
+
+    des_frames[0].data[0] = (FF_PCI_TYPE << PCI_TYPE_POS);
+    des_frames[0].data[1] = 63;
+    memcpy(&des_frames[0].data[2], random_data, 63);
```

Comment:
Should this not be 62 bytes only? Otherwise we overflow the `data` array.

```suggestion
    memcpy(&des_frames[0].data[2], random_data, 62);
```